src/lib/y2network/interface.rb

```diff
@@ -21,12 +21,17 @@ module Y2Network
   class Interface
     # @return [String] Device name (eth0, wlan0, etc.)
     attr_accessor :name
+    # @return [String]
+    attr_accessor :ethtool_options
```

Comment:
this is a kinda strange as only eth can have this options.

Reply:
btw there is also related to interface kernel module and its options. ( that is used for given interface )